src/main/java/com/gtbr/vaultbot/model/UserGuild.java

```diff
@@ -0,0 +1,29 @@
+package com.gtbr.vaultbot.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Table
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserGuild {
+
+    @Id
+    @Column
+    private String userGuildId;
+
+    @OneToOne
+    @JoinColumn(name = "guild_Id")
+    private Guild guildId;
```

Comment:
this attribute should be named as "guild" because we will access it as an object